osu.Game/Screens/Play/KeyCounter.cs

```diff
@@ -25,6 +25,11 @@ public abstract class KeyCounter : Container
         private readonly List<KeyCounterState> states = new List<KeyCounterState>();
         private KeyCounterState currentState;
 
+        [Resolved(canBeNull: true)]
+        private GameplayClock gameplayClock { get; set; }
+
+        public double CurrentTime => gameplayClock?.CurrentTime ?? Clock.CurrentTime;
```

Comment:
why public?

Reply:
Was used to `Clock.CurrentTime`..., anyways will remove.